lib/modules/datasource/conda/schema/prefix-dev.ts

```diff
@@ -0,0 +1,31 @@
+import { z } from 'zod';
+
+export const File = z.object({
+  version: z.string(),
+  createdAt: z.string().nullable(),
```

Comment:
you should have used the maybe timestamp here

Reply:
Oh, right. forget MaybeTimestamp is already a zod schema.

Reply:
please open a follow up PR with these changes 🙏

Reply:
ok, I'll send a new PR after https://github.com/renovatebot/renovate/pull/34646 and https://github.com/renovatebot/renovate/pull/34351 are merged.